src/main/java/step1/StringAddCalculator.java

```diff
@@ -5,32 +5,35 @@
 
 public class StringAddCalculator {
 
+    public static final String REGEX = "//(.)\n(.*)";
+    private static Pattern pattern = Pattern.compile(REGEX);
```

Comment:
```suggestion
    private static final Pattern pattern = Pattern.compile(REGEX);
```

static 변수의 경우 final 키워드를 붙여주는 것이 관례인데 어떠한 이유로 이러한 관례가 생겼는지 공부해보아요